### ISSUE TITLE
SqlServer

### ISSUE THREAD

**Issue by WillEhrendreich** [joe-re/sql-language-server#222]
Hey, I'm just curious if there are any plans to support MS SqlServer?

I use Sqls from neovim, but it's giving me a deprecation message and directing me to this Sqlls, but the one use case I need it for is not supported, hahaha. 

Does anyone know if this is being worked on? I couldn't even find a previous issue about it, is there really such a small demand for it?

**Comment by rcurran1221** [comment]
agreed - mssql server support is important to me, and no SQL LSP currently supports it! please save us SQL Server developers from needing to use VSCode/SSMS
